src/main/java/jalau/at18/kingoftokyo/DiceRoller.java

```diff
@@ -0,0 +1,39 @@
+package jalau.at18.kingoftokyo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DiceRoller {
+    private static final int COUNT_DICE = 6;
+    private RandomDice randomDice;
+
+    public DiceRoller() {
+        this.randomDice = new RandomDice();
+    }
+
+    //This method generated the list with random values.
+    public List<Dice> randomDice() { // No olvidar que aqui recibira la cantidad de randoms a generar a la lista
```

Comment:
in order to stick with instance/behavior logic, what does a DiceRoller do? roll dice, then, maybe `rollDice` would be a better name for this method. We know that dice will be returned with random results, but in terms of behavior, dice are rolled rather than "randomed"